libs/charts/src/filter/gen.rs

```diff
@@ -43,12 +43,12 @@
 //! [`get`]: fn.get.html (get function)
 //! [`set`]: fn.set.html (set function)
 //! [`set_from_cla`]: fn.set_from_cla.html (set_from_cla function)
-//! [`inactive`]: ./inactive (inactive module)
-//! [`alloc_site`]: ./alloc_site (alloc_site module)
+//! [`inactive`]: ./inactive/index.html (inactive module)
```

Comment:
I think you can remove these lines now, Rust builds the link `` [`inactive`] `` automatically if an item named `inactive` is in scope (a module, here).

Actually, most of the links around this one could be rewritten this way. Sometimes its a bit more verbose, like `` [`AllocSite`][alloc_site::AllocSite] `` assuming `AllocSite` is not in scope.